apps/website/src/content/docs/typography.mdx

```diff
@@ -20,12 +59,31 @@ thumbnail: Typography
 
 ## Label
```

Comment:
I'm not sure why `Label` was added to this page. Shouldn't we have a different page for it? It has nothing to do with typography.

Reply:
I guess it might be because `Label` was put under `Typography` in our v2 storybook as well. Should I separate it into another doc page?

Reply:
Yeah, but lets do it in a separate PR. We need to avoid breaking links, so it will involve three parts:
1. Create `text.mdx` and `label.mdx` pages with the respective docs
2. Keep `typography.mdx` page, but replace all the content with two links to the new pages
3. Remove `typography` link (and add `text` and `label` links) from sidebar and components page

Reply:
I have added this as a todo item.